### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by LaskoA**
I hope this one will be better :)

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -3,3 +3,59 @@ def __init__(self, weight):
...
+        self.coords[2] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+
+    def __init__(self, name, weight, max_load_weight,
+                 current_load, coords=None):
+        super().__init__(name, weight, coords)
+        if coords is None:
+            self.coords = [0, 0, 0]
```
Comment (Abnormaltype) [inline app/main.py:52]: that's not necessary

File: app/main.py [hunk 2]
```diff
@@ -3,3 +3,59 @@ def __init__(self, weight):
...
+    def get_info(self):
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+
+    def __init__(self, name, weight, coords=None):
+        super().__init__(name, weight, coords)
+        if coords is None:
+            self.coords = [0, 0, 0]
```
Comment (Abnormaltype) [inline app/main.py:37]: You can write this in one line)

File: app/main.py [hunk 3]
```diff
@@ -3,3 +3,59 @@ def __init__(self, weight):
         self.weight = weight
 
 # write your code here
+
+
+class BaseRobot:
+    def __init__(self, name, weight, coords=None):
+        if coords is None:
+            coords = [0, 0]
```
Comment (Abnormaltype) [inline app/main.py:11]: same here

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Good morning :)

**Review by Abnormaltype — APPROVED** [review]
Good job!

**Pull request closed without merging**